pkg/http/middleware/bodyfilter.go

```diff
@@ -0,0 +1,156 @@
+package middleware
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"path/filepath"
+
+	"golang.org/x/exp/slices"
+
+	"github.com/omissis/kube-apiserver-proxy/pkg/config"
+	http2 "github.com/omissis/kube-apiserver-proxy/pkg/http"
+)
+
+var ErrDuringBodyFilter = fmt.Errorf("error during body filter")
+
+func BodyFilterMux(conf []config.BodyFilterConfig) http2.MuxMiddleware {
+	return func(next http.Handler) http.Handler {
+		return BodyFilter(next, conf)
+	}
+}
+
+func BodyFilter(next http.Handler, conf []config.BodyFilterConfig) http2.Middleware {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		c, match := MatchConfig(r, conf)
+		if !match {
+			next.ServeHTTP(w, r)
+
+			return
+		}
+
```

Comment:
I would add a `slog.Debug()` call to print when a request is matched and when is not